setup.py

```diff
@@ -29,11 +29,11 @@
           'pandas>=0.23.0',
           'GitPython>=2.1.3',
           'xlsxwriter>=1.0.2',
-          'numpy>=1.12.1'
+          'numpy>=1.12.1',
+          'green>=2.13.0'
       ],
-      test_suite='nose.collector',
-      tests_require=['nose'],
       packages=find_packages(),
       include_package_data=True,
-      scripts=['bin/staramr']
+      scripts=['bin/staramr'],
+      tests_require = ['green']
```

Comment:
I took a quick look at what **tests_require** does. I don't think it does much unless you use the default Python testing packages. So, could i have you switch back to using **setup_requires** like how the `green` documentation specifies?  Sorry about that.

Reply:
No worries, changed it in 974b56e